Improve Ignore Order explanation. (#78)
quicwg/ack-frequency: draft-ietf-quic-ack-frequency.md

Ignore Order:

> : An 8-bit field representing a boolean truth value. This field can be

Reviewer: ```suggestion
: An 8-bit field representing a boolean truth value. This field is
```
